src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -336,6 +336,8 @@ namespace OpenMS
         peptides.back().setRT(f_it->getRT());
         peptides.back().setMZ(f_it->getMZ());
         peptides.back().setMetaValue("FFId_category", "internal");
+        peptides.back().setMetaValue("FromUntargeted", "true");
```

Comment:
do we need this? if there is a seed we should know that it is form untargeted?

Reply:
no but then I have to redo all tests

Reply:
Also, we never set FromUntargeted to "false" for the other features.
Maybe I am picky here but like this it doesn't really add value but just needs extra memory.

Reply:
I think we need a script that updates all failed tests ;)

Reply:
ok I will update them